src/server/openapi.yml

```diff
@@ -0,0 +1,446 @@
+  /services/tips/requestdataoverview:
+    get:
+      description: Endpoint not used on the client
+      responses:
+        '200':
+          description: Does things?
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/TipsRequestResponse'
+  /map/datasets/{datasetId}/{id}:
+    parameters:
+      - name: datasetId
+        in: path
+        required: true
+        description: The dataset id
+        schema:
+          type: string
+      - name: id
+        in: path
+        required: true
+        description: Some id
+        schema:
+          type: string
+    get:
+      description: Endpoint used to retrieve points of interest for the users area
+      responses:
+        '200':
+          description: Returns a list of Features
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MapDatasetResponse'
+  /status/cache:
+    get:
+      description: This endpoint gives an overview of the content inside the cache directory of the server.
+      responses:
+        '200':
+          description: Returns an object with cache overview
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CacheResponse'
+  /public/services/cms:
+    get:
+      description: Endpoint used to retrieve data from the CMS.
+      responses:
+        '200':
+          description: Returns an object with CMS data
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CmsResponse'
+  /public/services/cms/maintenance-notifications:
+    get:
+      description: Endpoint used to retrieve maintenance notifications.
+      responses:
+        '200':
+          description: Returns a list of CmsMaintenanceNotification objects
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/CmsMaintenanceResponse'
+  /direct/{apiName}:
+    parameters:
+      - name: apiName
+        in: path
+        required: true
+        description: The name of the api to request data from
+        schema:
+          type: string
+    get:
+      description: Retrieves data from the given api
+      responses:
+        '200':
+          description: Result is the direct response from the requested api
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ApiConfigResponse'
+components:
+  schemas:
+    HealthyResponse:
+      type: object
+      properties:
+        status:
+          type: string
+          enum:
+            - OK
+        content:
+          type: string
+          enum:
+            - OK
+    SearchConfigResponse:
+      type: object
+      properties:
+        staticSearchEntries:
+          oneOf: [$ref: '#/components/schemas/SearchEntry']

```

Comment:
Is er een specifieke reden waarom je `oneOf` hebt gebruikts? Ik modelleer dit normaal gesproken als:

```
staticSearchEntries:
  type: array
  items:
    $ref: '#/components/schemas/SearchEntry'
```

Reply:
oneOf wordt, volgens mij, met name gebruikt als je tussen verschillende objecten wilt differentiëren. Zie ook https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/